contracts/SSVToken.sol

```diff
@@ -0,0 +1,9 @@
+//SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.2;
+import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+
+contract SSVToken is ERC20 {
```

Comment:
this file shouldn't be here.
if you want to have a token for testing it you should create a `mocks` folder.

Reply:
moved